app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1400,6 +1408,13 @@ class BrowserTabViewModel @Inject constructor(
         }
     }
 
+    override fun onSitePermissionRequested(request: PermissionRequest) {
```

Comment:
are you going to add a test case for this method in this PR or in another one?

Reply:
They will be added in the 3rd PR: Site Permissions Settings. Next time I will try to add them sequentially 😬